onadata/apps/messaging/serializers.py

```diff
@@ -99,3 +99,22 @@ def create(self, validated_data):
                         "Message not created. Please retry.")
                 else:
                     return instance
+
+
+def send_mqtt_message(message, target_id, target_type, request):
+    """
+    Send an mqtt message.
+    :param message: message to send
+    :param target_id: id of the target_type
+    :param target_type: any of these three ['xform', 'project', 'user']
+    :param request: http request object
+    :return:
+    """
+    data = {
+        "message": message,
+        "target_id": target_id,
+        "target_type": target_type
+    }
+    message = MessageSerializer(data=data, context={"request": request})
+    if message.is_valid():
+        message.save()
```

Comment:
Does `message.save()` do multiple things i.e save and send message? If so, should the function name be changed to describe that the message is saved and sent?

Reply:
@ivermac the `save` function of the `MessageSerializer` class is not a custom method. It's the serializer's save method.